Makefile

```diff
@@ -101,9 +104,15 @@ client-ruby: $(wildcard clients/ruby/**/*)
 client-php: $(wildcard clients/php/**/*)
 	$(COMPOSER) install
 
-test: clients client-cpp client-js client-ruby client-php
+client-go: $(wildcard clients/go/**/*)
+	$(GO) get -u github.com/go-bindata/go-bindata/...
+	cd clients/go
+	$(GO) generate
+
+test: clients client-cpp client-js client-ruby client-php client-go
 	cd clients/cpp && $(MAKE) test
 	cd clients/java && $(MVN) test
 	cd clients/js && $(NPM) test
 	cd clients/ruby && $(RAKE) test
+	$(GO) get -u github.com/go-bindata/go-bindata/... && cd clients/go && $(GO) generate && $(GO) test
```

Comment:
This is already run in the `client-go` recipe, so you should need only `cd clients/go && $(GO) test`

Reply:
but when other people try to install it would expect it to work with just `go get`

Reply:
What I mean is, when you run `make test`, the `$(GO) generate` stuff is already executed in the `client-go` recipe, which is a dependency of the `test` recipe.  You should keep this in the `client-go` recipe, but you can remove it from the `test` recipe.

Reply:
@sffc ah sorry, will fix it now

Reply:
@sffc done